participants/Tanisha07.md

```diff
@@ -1,5 +1,7 @@
-Name: Tanisha Sharda
-GitHub username: Tanisha07
-University: IGDTUW
-Year: 2
-Branch: IT
+______________________________________
+ #### `**Name:** Tanisha Sharda`
```

Comment:
Hi @Tanisha07, the bold using ** does not work inside ``.

Reply:
Okay, yes. I realized this after viewing the file on StackEdit and made changes there but forgot to update those changes and on my file. I'll update it again.